book/src/dev/rfcs/xxxx-block-subsidy.md

```diff
@@ -0,0 +1,302 @@
+- Feature Name: Block subsidy
+- Start Date: 2020-10-05
+- Design PR: [ZcashFoundation/zebra#0000](https://github.com/ZcashFoundation/zebra/pull/0000)
+- Zebra Issue: [ZcashFoundation/zebra#0000](https://github.com/ZcashFoundation/zebra/issues/0000)
+
+# Summary
+[summary]: #summary
+
+
+Zebra manages [semantic verification](https://github.com/ZcashFoundation/zebra/blob/main/book/src/dev/rfcs/0002-parallel-verification.md#definitions) in the `zebra-consensus` crate, this is done for all incoming blocks. Inside each block the coinbase transaction is special, it holds the subsidy rewards that are paid to different participants (miners, founders, funding stream receivers). This RFC describes how to implement the needed calculations and verification for block subsidy and miner fees.
+
+# Motivation
+[motivation]: #motivation
+
+All incoming blocks must be validated with the protocol specifications, there is no way to avoid including this logic in any Zcash protocol compatible implementation such as Zebra.
+
+Block subsidy and miner fees are part of the protocol, the semantic verification of them apply to the coinbase transaction and they must be verified in Zebra.
+
+# Definitions
+[definitions]: #definitions
+
+- **founders reward**: The portion of the block reward that goes into a pre defined founder address. 
+- **funding streams**:  The portion of the block reward that goes into a pre defined funding stream address.
```

Comment:
These definitions are subtly incorrect. A certain part of the block subsidy is required to go to a Founders' Reward address and/or to one or more funding stream addresses. But that's not necessarily equal to the amount that actually goes to those addresses, in coinbase transactions or otherwise. In particular the miner is not prevented from sending additional outputs to those addresses.